frontend_tests/node_tests/search_suggestion.js

```diff
@@ -573,6 +574,7 @@ run_test('check_is_suggestions', () => {
     var expected = [
         'i',
         'is:private',
+        "is:public-stream",
```

Comment:
We probably want this to be single quoted. I'm surprised that the linter didn't catch this, though.

Reply:
Same with a few other equivalent changes below. Just for the sake of consistency.

Reply:
Done